src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -430,4 +440,86 @@ describe('Test suite for dictionary concepts components', () => {
     expect(mapStateToProps(initialState).loading).toEqual(false);
     expect(mapStateToProps(initialState).dictionaries).toEqual([]);
   });
+
+  it('should update the state with search term on change in the search bar', () => {
+    const props = {
+      match: {
+        params: {
+          typeName: 'dev-col',
+          type: 'orgs',
+          collectionName: 'dev-col',
+          dictionaryName: 'dev-col',
+        },
+      },
+      location: {
+        pathname: '/random/path',
+      },
+      fetchDictionaryConcepts: jest.fn(),
+      concepts: [],
+      filteredClass: ['Diagnosis'],
+      filteredSources: ['CIEL'],
+      loading: false,
+      conceptsCount: 1,
+      totalConceptsCount: 1,
+      filterBySource: jest.fn(),
+      filterByClass: jest.fn(),
+      fetchMemberStatus: jest.fn(),
+      paginateConcepts: jest.fn(),
+      totalConceptCount: 20,
+      userIsMember: true,
+      removeDictionaryConcept: jest.fn(),
+      removeConceptMappingAction: jest.fn(),
+      searchByName: jest.fn(),
+    };
+    const wrapper = mount(<Provider store={store}>
+      <Router>
+        <DictionaryConcepts {...props} />
+      </Router>
+    </Provider>);
+    let event = { target: { name: 'searchInput', value: 'testing' } };
+    wrapper.find('#search-concept').simulate('change', event);
+    expect(wrapper.find('DictionaryConcepts').state().searchInput).toEqual('testing');
+    event = { target: { name: 'searchInput', value: '' } };
+    wrapper.find('#search-concept').simulate('change', event);
+    expect(wrapper.find('DictionaryConcepts').state().searchInput).toEqual('');
+  });
+
+  it('should call the search function when search form is submitted', () => {
+    const props = {
+      match: {
+        params: {
+          typeName: 'dev-col',
+          type: 'orgs',
+          collectionName: 'dev-col',
+          dictionaryName: 'dev-col',
+        },
+      },
+      location: {
+        pathname: '/random/path',
+      },
+      fetchDictionaryConcepts: jest.fn(),
+      concepts: [],
+      filteredClass: ['Diagnosis'],
+      filteredSources: ['CIEL'],
```

Comment:
Don't we already have a constant for this?

Reply:
No we don't and I didn't want to refactor the test file in this PR.

Reply:
Are you sure? Have you asked your team mates if it does not already exist?

Reply:
@dkayiwa let me ask my teammates although from observation I can see that the `const props` was scoped to the individual `it()` functions.

Reply:
@dkayiwa I think I hadn't understood your question. After consulting the team, I have made necessary changes.